styles.css

```diff
@@ -0,0 +1,1007 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300&family=Poppins:wght@100&family=Roboto&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  width: 1100px;
+  margin-left: 100px;
+  background-color: #e5e5e5;
+}
+
+.grid {
+  display: grid;
+  grid-template-rows: 20% 40% 23% 17%;
+}
+
+.home {
+  color: #fff;
+  width: 1100px;
```

Comment:
I agree that specifying width changes the properties and appearance of the project, but I have to change that because it is a requirement of the project. I am attaching a screenshot: 
![requirement](https://user-images.githubusercontent.com/52704856/145531788-92d99043-35d5-4061-9995-4526f2819336.png)

Reply:
The `width` of `1100px` should be implemented in your CSS media query. It simply means that your app changes to desktop view when the viewport is more than **992** or **1100** px.
You may want to [read this about media queries.](https://www.w3schools.com/css/css_rwd_mediaqueries.asp)

Reply:
@Gegardus, Your media query has issues. Please refer to your project requirement and make sure you understand it. You can leverage your morning session group, coding partners, or the standup group to understand how to use media queries and the project requirement.
![image](https://user-images.githubusercontent.com/45668340/145543883-77657031-f1c9-4d8f-a7ea-b04bad86ade1.png)

Reply:
Thank you. But I do not completely understand, do you mean that  I should have two different designs, and when it goes after 1100 width one design should change the previous design?

Reply:
Yes. That's what the media query does. The first CSS codes applies to the mobile version but all other CSS codes inside the media query rule apply to the desktop. It is a sort of having different CSS codes for the two versions but there will definitely be other CSS rules that will be maintained from that of mobile in the desktop viewport. Follow the link I dropped for media queries to get a better grasp of how it works. You can contact me for a video chat if you want.

Keep up the good work👍

Reply:
Well, I am really confused. There are not only different CSS rules but also completely different HTML construction. And CSS rules are almost all different. To implement this, I should have two different HTML and CSS files, that will be swapped. How I can put this all in a media query?

Can you kindly give me further explanations?

Reply:
_Example_
```
/* Mobile version CSS */
.class1 {
    color: red;
}

.class2 {
    color: green;
}

.class3 {
    color: white;
}

/* Desktop version CSS */
@media screen and (min-width: 992px) {
    .class2 {
        color: skyblue;
    }

    .class3 {
        color: lightgreen;
    }
}
```
In the above example, I have only one CSS file but I am telling the computer to apply the first code for screens that are from 0 to 992 pixels wide(mobile).

Then i am telling it to change some styles when the screen width is wider than 992 pixels(desktop).

This is how media query works.

Reply:
Done.